src/manifest.ts

```diff
@@ -404,6 +405,43 @@ export class Manifest {
     }
 
     const needsBootstrap = releasesFound < expectedReleases;
+    if (releasesFound < expectedReleases) {
+      logger.warn(
+        `Expected ${expectedReleases} releases, only found ${releasesFound}`
```

Comment:
This adds quite a bit of logic, I think it would be worth pulling it into a helper method, just to encapsulate it.

What state does the repo need to be in for this additional lookup to be necessary, tags without releases?